sov-modules/sov-modules-macros/tests/module_info/missing_serialization.stderr

```diff
@@ -0,0 +1,5 @@
+error: The `ModuleInfo` macro supports structs only.
+ --> tests/module_info/derive_on_enum_not_supported.rs:5:1
+  |
+5 | enum TestStruct {
+  | ^^^^
```

Comment:
Can you help me with the test, how should I invoke it?

Reply:
Yeah, so we use `trybuild` crate for testing the macros. The tests are run automatically as part of `cargo test` in `sovereign/`. If you `cargo test` in `sov-modules/sov-modules-marcros`, you will see **only** the macro-related tests (so I recommend this).  Check `all_tests.rs` to see how to add new tests. If the output (`.stderr` file) of the tests changes `trybuild` will ask you to update the file (just follow the steps it prints to the the terminal)

Reply:
Are you sure pipelines work correctly as this PR is green, while this test should fail?

Reply:
It looks like the the test was not added to `all_tests.rs`. Also let's move it to `tests/dispatch` folder. `module_info` contains tests for `ModuleInfo` macro but the attribute was added to `Dispatch..` macro